sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTest.java

```diff
@@ -573,6 +579,9 @@ public void getKeyRotationPolicyWithNoPolicySet(HttpClient httpClient, KeyServic
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("getTestParameters")
     public void updateGetKeyRotationPolicyWithMinimumProperties(HttpClient httpClient, KeyServiceVersion serviceVersion) {
+        if (!isPublicCloud() && interceptorManager.isLiveMode()) {
```

Comment:
Is there a way to mark these tests as skipped rather than showing that they pass with empty operation? I see the use of `Assumptions.assumeTrue` below, perhaps that's a better option, but I'll let a java sdk dev weigh in.